infra/modules/providers/azure/function-app/README.md

```diff
@@ -255,14 +72,21 @@ You will need to provide the following information for the dependencies:
 
 This is a map where the key is the `function_app_name` and the definition for the function app:
 
-  - **`app_settings`:** The function app settings config map.
-  - **`image`:** The docker image name.
+- For Docker based deployment, the object has one field:
+image: which refers to the docker image name to deploy.
+- For running from a package, it should contains the fields:
+  - zip: contains an http reference to the package.
+
+> This will enable your function app to run from a package by adding a WEBSITE_RUN_FROM_PACKAGE setting to your function app settings.
+
+- hash: contains a hash of the zip file for downloads integrity check.
+
 
 ### Manually deploying Private ACR Images
 
 If no image is deployed due to the usage of a Private ACR configuration then the `image` should be left blank and images can be deployed post terraform provisioning has occured by using the Azure CLI or some other process.
```

Comment:
In addition to the important explanation here, we can be be a bit more clear and say something like, `"..or some other process. The below example uses the Azure CLI and docker."`